pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -40,10 +39,6 @@ func (c *Cluster) makeDeployment(mgrConfig *mgrConfig) *apps.Deployment {
 				"prometheus.io/port": strconv.Itoa(metricsPort)},
 		},
 		Spec: v1.PodSpec{
-			InitContainers: []v1.Container{
```

Comment:
Nit: The convention I have been following with the other daemons is to set `InitContainers` explicitly to `[]v1.Container{}` for some extra clarity to devs that there are no init containers for the daemon.

Reply:
Fixed